users/middlewares.go

```diff
@@ -1,12 +1,13 @@
 package users
 
 import (
-	"github.com/dgrijalva/jwt-go"
-	"github.com/dgrijalva/jwt-go/request"
-	"github.com/wangzitian0/golang-gin-starter-kit/common"
-	"gopkg.in/gin-gonic/gin.v1"
 	"net/http"
 	"strings"
+
+	jwt "github.com/dgrijalva/jwt-go"
```

Comment:
Same here. Codes LGTM, but please make sure it works fine in new environment.

Reply:
OK, make sure later on.